calico-vpp-agent/policy/policy_server.go

```diff
@@ -84,7 +84,9 @@ type Server struct {
 	workloadsToHostPolicy *Policy
 	workloadAddresses     map[string]string
 	/* always allow traffic coming from host to the pods (for healthchecks and so on) */
```

Comment:
We should probably be more descriptive in the comment & the function creating it.
e.g.
````golang
// allowFromHostPolicyId persist the ID of the policy allowing host <-> pod communications. 
// See CreateAllowFromHostPolicyId definition
allowFromHostPolicyId uint32
````
and 
````golang
// CreateEgressAllowFromHostPolicy creates a policy allowing host->pod communications. This is needed
// to maintain vanilla Calico's behavior where the host can always reach pods on any pods.
// this policy is applied in Egress on the host endpoint tap (i.e. linux -> VPP)
// and on the Ingress of Workload endpoints (i.e. VPP -> pod)
func CreateEgressAllowFromHostPolicy() { ... }
````